### DISCUSSION TITLE
Where did the file history button go?!!!

### DISCUSSION THREAD

**Discussion by babakfp** [gitkraken/vscode-gitlens#2948] [Q&A] [answered]
Hey

https://github.com/gitkraken/vscode-gitlens/assets/44144724/90080054-c1ee-4be1-a55f-ca1862cac4a1

Back then I could click on that button again and again but now it's not possible!

**Comment by Roberto-Vizcarra** [comment, marked as answer]
Hello,

This usually happens when there are too many icons in that toolbar. In order to open it, you can just click the ellipsis icon "..." and choose "Open Changes with revision history". If you would like to keep it there, you can disable other icon in that toolbar by right clicking on one of the icons and unclicking some of the other icons. This should make enough room to move the icon back into view.

![image](https://github.com/gitkraken/vscode-gitlens/assets/98343055/e85e8818-d614-4e47-aebc-5840d1ec8501)

**Reply by babakfp** [reply]
Hi

Thanks, it fixed it. However there is plenty of space, and this behavior feels buggy and I was missing a very useful feature, until I asked about it, other users may not be so lucky and they may feel frustrated like me!

![Screenshot 2023-10-24 173257](https://github.com/gitkraken/vscode-gitlens/assets/44144724/0a2021c5-3091-456d-877d-472508b20800)

**Reply by eamodio** [reply]
Unfortunately we have no control over that -- VS Code imposes that limit and we can't change it/do anything about it.

**Reply by babakfp** [reply]
Is it possible to change the priority on which one is going to stay visible/get hidden? Maybe that can work and it can be a good workaround 🤷.

**Reply by eamodio** [reply]
Unfortunately not really. It's kind of the Wild West for ordering and gets really hard with multiple extensions (built in and external) all contributing to the same bar
